tasks.rb

```diff
@@ -446,7 +450,8 @@ def create_hpxmls
     'base-simcontrol-timestep-10-mins-occupancy-stochastic-10-mins.xml' => 'base-simcontrol-timestep-10-mins.xml',
     'base-simcontrol-timestep-10-mins-occupancy-stochastic-60-mins.xml' => 'base-simcontrol-timestep-10-mins.xml',
     'base-simcontrol-timestep-30-mins.xml' => 'base.xml',
-    'base-vacancy.xml' => 'base.xml'
+    'base-vacancy.xml' => 'base.xml',
+    'base-power-outage.xml' => 'base.xml'
```

Comment:
Do we really need this? Seems like it doesn't test anything different than base-schedules-simple-power-outage.xml.

Reply:
I was just mimicking the vacancy sample files...

Reply:
I deleted them.